lib/LaTeXML/Package/enumitem.sty.ltxml

```diff
@@ -66,6 +66,10 @@ DefKeyVal('enumitem', 'beginpenalty', 'Number');
 DefKeyVal('enumitem', 'midpenalty',   'Number');
 DefKeyVal('enumitem', 'endpenalty',   'Number');
 DefKeyVal('enumitem', 'noitemsep',    'UndigestedKey');
+DefKeyVal('enumitem', 'nolistsep',    'UndigestedKey');
```

Comment:
I think these should just be
```
DefKeyVal('enumitem','noitemsep','','true');
```
which corresponds better to enumitem's code; the 4th arg is the default.

Reply:
Alright, changed now.